homeassistant/components/inels/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+  "domain": "inels",
+  "name": "iNELS",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/inels",
+  "requirements": ["inels-mqtt==0.0.35"],
```

Comment:
The metadata of the module are outdated and `paho-mqtt` is pinned to 1.6.1.

Reply:
Please be more specific. 

Inels-mqtt module is latest release, so what do you mean metadata is outdated? 

And, yes paho-mqtt is pinned to 1.6.1 as same as for example mqtt integration.

Reply:
> Inels-mqtt module is latest release, so what do you mean metadata is outdated?

https://github.com/Nejezchleb/inels-mqtt is 404

> And, yes paho-mqtt is pinned to 1.6.1 as same as for example mqtt integration.

At the moment is Home Assistant using `paho-mqtt-1.6.1`. The module should have a constrain if it doesn't work with 1.6.2 otherwise the pinning should be removed.

Reply:
> https://github.com/Nejezchleb/inels-mqtt is 404

Module was in private repo. I change it into public

> The module should have a constrain if it doesn't work with 1.6.2 otherwise the pinning should be removed.

I'll change it into `paho-mqtt>=1.6.1`. Will be released in version `inels-mqtt==0.0.36`